sovrin_client/agent/walleted.py

```diff
@@ -97,6 +103,17 @@ def __init__(self,
         }
         self.logger = logger
 
+        self.issuer_backend = None
+
+        self._invites = {}  # type: Dict[Nonce, Tuple(InternalId, str)]
+        self._attribDefs = {}  # type: Dict[str, AttribDef]
+        self.defined_claims = []  # type: List[Dict[str, Any]
+
+        self.available_claims_by_internal_id = {}  # type: Dict[InternalId, Set[ID]]
```

Comment:
In latest master branch code, few attributes (like self._proofRequestsSchema and few other to hold available claims for user) which were specific to issuer wallet only, has been moved to SovrinIssuerWalletInMemory class (in sovrin-client/sovrin_client/anon_creds/sovrin_issuer.py file), but I do see it here as well.